server/database/queries/getAppointmentsByIdQuery/index.js

```diff
@@ -0,0 +1,11 @@
+const connection = require('../../config/connection');
+
+const getAppointmentsByIdQuery = ({ appointmentId }) => {
```

Comment:
You here get only the is_done from the appointment record,
I think you need to change the name of this query to be getAppointmentStatus.
And also you don't need to select the appointment id

Reply:
done